addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -276,7 +276,7 @@
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'base',
-                    'plus_report_line_ids': [ref('l10n_de.tax_report_de_tag_41')],
+                    'plus_report_line_ids': [ref('l10n_de.tax_report_de_tag_41'), ref('l10n_de.tag_de_intracom_community_delivery')],
```

Comment:
plus_report_line_ids is made to reference account.tax.report line objects; tag_de_intracom_community_delivery is an account.account.tag ;) For cases of cross-reference to a financial report using account.account.tag directly like here, you can use the tag_ids field instead of plus/minus_report_line_ids, like in this example: https://github.com/odoo/odoo/blob/14.0/addons/l10n_es/data/account_tax_data.xml#L265

So here, just to be clear, only add a tag_ids key with tag_de_intracom_community_delivery, and keep the rest as-is :)

Reply:
@oco-odoo Thanks for the explanation! My bad I was a bit a bit inatentive for this one